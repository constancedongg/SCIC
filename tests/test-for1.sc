int func main() {
    int[] arr = [1, 3, 5, 7, 5, 3, 1];
    int i = 0;
    for (; i < 7; i = i + 1) {
        print(arr[i]);
        arr[i] = i;
    }

    printl("========");

    for (i = 6; i >= 0; i = i - 1) {
        print(arr[i]);
    }
    return 0;
}