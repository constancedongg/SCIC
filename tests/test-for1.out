1
3
5
7
5
3
1
========
6
5
4
3
2
1
0